packages/plugin-session-replay-browser/package.json

```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "@amplitude/analytics-types": ">=1 <3",
     "idb-keyval": "^6.2.1",
-    "rrweb": "^2.0.0-alpha.4",
+    "rrweb": "^2.0.0-alpha.9",
```

Comment:
Alpha worries me a little lol.  Should we be using the stable 1.x version instead? what does datadog use?

Reply:
Yeah I was wondering about that too... I'll see what the others are using. All of the docs are referencing the most recent alpha version (I only updated to this because the maskInputFn signature did not match the docs, so it broke and made me realize we weren't using the latest package), so I wonder how much we'd need to change to go to 1.x